tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -2775,4 +2777,28 @@ public function getPermissionRoleDefinitionUsingGraphAPI(string $user, string $p
 			)
 		);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" checks the activities for (folder|file) "([^"]*)" in space "([^"]*)"/
+	 *
+	 * @param string $user
+	 * @param string $fileOrFolder   (file|folder)
+	 * @param string $resource
```

Comment:
no use for `$fileOrFolder`
```suggestion
	 * @When /^user "([^"]*)" checks the activities for (?:folder|file) "([^"]*)" in space "([^"]*)"/
	 *
	 * @param string $user
	 * @param string $resource
```